src/main/resources/application.yml

```diff
@@ -37,24 +45,27 @@
 
   services:
     persistence:
+      baseUrl: http://localhost:8282
       #service.url:  ${probate_persistence_service_url}:${probate_persistence_service_port}/formdata  #Till byron fixed side way communcation of app
-      formdata.url:  http://localhost:8282/formdata
+      formdata.url:  ${services.persistence.baseUrl}/formdata
       #service.url:  ${probate_persistence_service_url}:${probate_persistence_service_port}/submissions  #Till byron fixed side way communcation of app
-      submissions.url:  http://localhost:8282/submissions
-      sequenceNumber.url:  http://localhost:8282/sequence-number
+      submissions.url:  ${services.persistence.baseUrl}/submissions
+      sequenceNumber.url:  ${services.persistence.baseUrl}/sequence-number
 
     coreCaseData:
-      url: https://case-data-app.dev.ccd.reform.hmcts.net:4451/citizens/%s/jurisdictions/PROBATE/case-types/GrantOfRepresentation
+      baseUrl: https://case-data-app.dev.ccd.reform.hmcts.net:4451
```

Comment:
Check references to the property

Reply:
This change should be ok. The property services.coreCaseData.url still exists in the application.yml file. For each CNP environment the services.coreCaseData.baseUrl would need to change so the health endpoint and ccd service can be reached. Praveen is aware of this.